### PR TITLE
Basic dashboard + school list

### PR DESCRIPTION
**Pull request by felipereigosa**

### CODE REVIEW

File: src/cljs/webchange/admin/schools/views.cljs [hunk 1]
```diff
@@ -0,0 +1,65 @@
+(ns webchange.admin.schools.views
+  (:require [re-frame.core :as re-frame]
+            [webchange.editor-v2.components.card.views :refer [list-card] :as card]
+            [cljs-react-material-ui.reagent :as ui]
```
Comment (wildeastengineer) [inline src/cljs/webchange/admin/schools/views.cljs:4]: do not use material-ui. we are going to get rid of it
use components from webchange/ui_framework/components/index.cljs instead

File: src/cljs/webchange/admin/schools/views.cljs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            [cljs-react-material-ui.icons :as ic]))
+
+(re-frame/reg-sub
```
Comment (wildeastengineer) [inline src/cljs/webchange/admin/schools/views.cljs:7]: state should be in separate file state.cljs

Comment (wildeastengineer) [inline src/cljs/webchange/admin/schools/views.cljs:7]: state should be in separate file state.cljs

File: src/cljs/webchange/admin/schools/views.cljs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+
+(defn school-item [data]
+  (let [list-styles (card/get-styles)]
+    (println data)
+    [ui/list-item
+     [ui/list-item-text {:primary (:name data)}]
+     [ui/list-item-secondary-action
+      [ui/button {:aria-label "Presentation"
+                  :style {:background-color "white"
```
Comment (wildeastengineer) [inline src/cljs/webchange/admin/schools/views.cljs:28]: don't use inline styles. component styles should be defined in styles.scss file

Comment (wildeastengineer) [inline src/cljs/webchange/admin/schools/views.cljs:28]: don't use inline styles. component styles should be defined in styles.scss file

Comment (felipereigosa) [reply src/cljs/webchange/admin/schools/views.cljs:28]: Yes, I'm aware some code should be in state.cljs and styles.scss, that's why I created the schools/styles.scss even though I'm not using it. This code was a prototype, Ivan asked me to create a draft with what I had at the end of the day even though it wasn't finished.

**Pull request closed without merging**